src/components/cc-pricing-estimation/cc-pricing-estimation.js

```diff
@@ -149,9 +149,17 @@ export class CcPricingEstimation extends LitElement {
    * @return {string|void} the translated feature name if a translation exists or nothing if the translation does not exist
    */
   _getFeatureName (feature) {
-    if (feature != null && FEATURES_I18N[feature.code] != null) {
+    if (feature == null) {
+      return '';
+    }
+
+    if (FEATURES_I18N[feature.code] != null) {
       return FEATURES_I18N[feature.code]();
     }
+
+    if (feature.name != null) {
+      return i18n('cc-pricing-estimation.feature.custom', { featureName: feature.name });
+    }
```

Comment:
Maybe we should always try to return something instead of `void`: use `feature.code` as a fallback.

```suggestion
    return i18n('cc-pricing-estimation.feature.custom', { featureName: feature.name ?? feature.code });
```

Reply:
The issue is that the `cc-pricing-estimation` filters features that have no `name` or no translation registered for a given feature code.

```js
plan.features
  ?.filter((feature) => AVAILABLE_FEATURES.includes(feature.code) || feature.name != null)
  ?.map((feature) => {
    return html`
      <div class="plan__features__feature">
        <dt>${this._getFeatureName(feature)}</dt>
        <dd>${this._getFeatureValue(feature)}</dd>
      </div>
    `;
})
```
To make this change useful, we'd have to remove the filter and let everything go through.
But to be able to do this, the `cc-pricing-product` would need to clean the features it sends to the `cc-pricing-estimation` and remove the ones that are not part of `productFeatures`. 

We discussed this refactoring (plan features model rework + `cc-pricing-product` not relying only on its smart component to filter features) with Hubert and Mathieu and decided not to go down this road for now.

The main reason is that this is only useful when the component is not used with a smart component, which is very rare (only for 1 or 2 products on the website, where they will make sure to use `name` properties anyway).